src/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  letter-spacing: 1px;
+  background-color: #d5cdcd;
+}
+
+.container {
+  width: 40%;
+  margin: 5%  auto;
+  background-color: white;
+  box-shadow: rgba(75, 71, 71, 0.836) 0 8px 24px;
+  border-radius: 2%;
+}
+
+.clear-btn {
+  width: 60%;
+  margin: auto;
+}
+
+.input-form {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+  padding: 20px 10px;
+}
+
+.input-form .input-task {
+  border: none !important;
```

Comment:
Try to avoid using `!important` in your CSS. Using `!important`, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets.